app/controllers/application_controller.rb

```diff
@@ -154,6 +154,10 @@ def delete_session
     flash[:confirmation] = 'Session will expired'
   end
 
+  def destroy_session
+    flash[:session_destroyed] = 'Session removed'
```

Comment:
The actual content of the flash is moot as it's never shown, we just redirect if it's present.